src/main/index.js

```diff
@@ -1,137 +1,135 @@
 import { app, BrowserWindow, ipcMain as ipc } from 'electron';
 import localShortcut from 'electron-localshortcut';
 import browserLauncher from '@james-proxy/james-browser-launcher';
-import Raven from 'raven';
+import * as Sentry from '@sentry/node';
 
 import constants from 'common/constants.js';
 import config from 'common/config.js';
-import ravenInit from 'common/service/raven.js';
+import sentryInit from 'common/service/sentry.js';
 
 import createMenu from './menu.js';
 import createUrlMapper from './url-mapper.js';
 import createProxy from './proxy.js';
 import autoUpdater from './auto-update.js';
 
-ravenInit(Raven);
-Raven.context(() => {
-  // Keep a global reference of the window object, if you don't, the window will
-  // be closed automatically when the javascript object is GCed.
-  let mainWindow = null;
+sentryInit(Sentry);
+// Keep a global reference of the window object, if you don't, the window will
```

Comment:
Pretty much all of this is just formatting since we didn't need the outside `Raven.context(...)` anymore